_data/infrastructure-projects/police-stations-hash147.yaml

```diff
@@ -0,0 +1,34 @@
+name: Police Stations
+coordinates:
+  - -30.4759, 30.2030
+stage: Various
+total_budget: 142200000000
+department: Police
+description: Construction of new and re-established police stations
+projected_budget: 51800000000
+department_link: /path/to/department/detail/page
```

Comment:
let's use `url` rather than `link` to be consistent with the rest of the data and let's put the name and URL in a dictionary like in the overview